### ISSUE TITLE
Feature Request: Limit clipboard size

### ISSUE THREAD

**Issue by VeRychard** [PasteBar/PasteBarApp#119]
However it's great that there can be limitless clips, it would make sense to have an option to limit it in the settings.
Either for size of the "db" or the number of entries, or auto-delete

![image](https://github.com/user-attachments/assets/fc155883-0064-4f68-a9f8-7f64ba9e1269)

**Comment by kurdin** [comment]
@VeRychard  I have an option in clipboard history settings where you can set auto clear duration in days, weeks, months, years

<img width="656" alt="image" src="https://github.com/user-attachments/assets/1c63c73b-7b7b-432d-bd33-a375d55ac87b">

I think it could be useful to have a max number of saved history items as well so I'll add this feature in the future. 
Thank you @VeRychard !

**Comment by VeRychard** [comment]
You are right for the time part. :) So that setting could have another field for size in GB and/or number of entries 👌
